docs/seeding.md

```diff
@@ -1,6 +1,6 @@
 ## What Do Seeders Do?
 
-Seeders canvass the resources of a given government agency, identifying important URLs and whether those URLs can be crawled by the Internet Archive's web crawler. They use the [EDGI Nomination Chrome extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en) to nominate URLs to the [End of Term (EOT) Web Archive](http://eotarchive.cdlib.org/2016.html) if they are crawlable or to the Archivers app if they require manual archiving.
+Seeders canvass the resources of a given government agency, identifying important URLs and whether those URLs can be crawled by the Internet Archive's web crawler. They use the [EDGI Nomination Chrome extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en) to nominate URLs to the [Internet Archive](http://www.archive.org) if they are crawlable or to the Archivers app if they require manual archiving.
```

Comment:
I think we might need to update the "nominate" language as well, something like "save URLS to the Internet Archive" (as that is the terminology IA uses).